python/featomic/tests/clebsch_gordan/equivariant_power_spectrum.py

```diff
@@ -136,6 +136,41 @@ def test_equivariant_power_spectrum_neighbors_to_properties():
     metatensor.equal_raise(powspec_1, powspec_2)
 
 
+def tests_sample_selection() -> None:
```

Comment:
Just wondering if this would be run by pytest if not called `test_*` @Luthaf ?
```suggestion
def test_sample_selection() -> None:
```

Reply:
ah, no it will not run. `pytest` needs the function to be named `test_whatever`

Good catch!